package.json

```diff
@@ -12,11 +12,8 @@
     "docs:build": "compodoc -p --tsconfig ./tsconfig.json -n ngx-device-detector -d docs --hideGenerator --theme Vagrant",
     "docs:serve": "npm run docs:build -- -s",
     "docs:watch": "npm run docs:build -- -s -w",
-    "lint:demo": "prettier --check projects/demo/src/**/*.ts",
-    "lint:lib": "prettier --check projects/ngx-device-detector/src/**/*.ts",
-    "lint:lib:fix": "prettier --write projects/ngx-device-detector/src/**/*.ts",
-    "lint": "npm run lint:lib",
-    "lint:fix": "npm run lint:lib:fix",
+    "format:check": "prettier --config ./.prettierrc --list-different \"{projects,src}/**/*{.ts,.js,.json,.html,.scss}\"",
+    "format:all": "prettier --config ./.prettierrc --write \"{projects,src}/**/*{.ts,.js,.json,.html,.scss}\"",
```

Comment:
I think the build is broken because of the missing lint scripts.
Just add the scripts back:
`    "lint:demo": "prettier --check projects/demo/src/**/*.ts",
    "lint:lib": "prettier --check projects/ngx-device-detector/src/**/*.ts",
    "lint:lib:fix": "prettier --write projects/ngx-device-detector/src/**/*.ts",
    "lint": "npm run lint:lib",
    "lint:fix": "npm run lint:lib:fix",`

Reply:
Hello @Jorciney, I fixed the scripts, the lint script was just calling the format check on the lib.